pages/checkout/terms.tsx

```diff
@@ -65,7 +77,14 @@ const CheckoutTermsView: NextPage = () => {
         </IngressContainer>
 
         {/* TODO: Add banner */}
-
+        <BannerContainer>
+          <Banner variant="danger">
+            <Title>{intl.formatMessage({ id: "terms.banner.danger.heading1" })}</Title>
```

Comment:
Add some semantics to the Title here using the Typography component. It's always important to use the Typography component as it adds Equinor styles to the text